ISISComputingGroup/JSON_bourne
# Ticket2509: Overview status screen

See: https://github.com/ISISComputingGroup/IBEX/issues/2509
Also adds favicon, see: https://github.com/ISISComputingGroup/IBEX/issues/1677

**To test**
- Check that instrument page and overview page have favicon
- Test clicking on an instrument button will open a window underneath with the instrument details
- Test that if an instrument goes down, the result is that the corresponding button turns red
- Test that the x closes the window below the buttons
- Check the links at the top of the pages take you to the correct webpages
